lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -72,6 +73,16 @@ function requestHandler(request, response) {
         delay = parseInt(queryString.delay, 10) || 2000;
       }
 
+      if (typeof queryString.extra_header !== 'undefined') {
+        let extraHeaders = queryString.extra_header;
+        extraHeaders = Array.isArray(extraHeaders) ? extraHeaders : [extraHeaders];
+
+        extraHeaders.forEach(header => {
+          const parts = header.split(':');
```

Comment:
I actually would prefer `const [key, ...value]`, but that's just down to preference.

You could also do `header.split(/:(.+)/);`, which should give the correct split (captured groups also appear in the resulting array)